### PR TITLE
this  forwardref  for the select.

### PR DESCRIPTION
**Pull request by Ayush2020012016**
- [x] Added the forward ref for the select.
- [x] Added the documentation for the select.

### CODE REVIEW

File: components/select-dropdown/src/ReuseSelectDropdown.tsx [hunk 1]
```diff
@@ -217,9 +217,10 @@ const ReuseSelectDropDown: React.FC<ReuseSelectProps> = ({
       onChange={(e) => setQuery(e.target.value)}
     />
   );
-
+console.log("this is form the select base file.",{ref})
```
Comment (ShreyDhyani) [inline components/select-dropdown/src/ReuseSelectDropdown.tsx:220]: Remove this log

File: development/select-dropdown-app/components/select-dropdown/ForwardRefSelectDropdown/ForwardRefSelectDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        enableSearch
+        refresh={refresher}
+      />
+    </div>
+  );
+};
+
+export default ForwardRefInputGroup;
+
+// optionsRenderer={(
```
Comment (ShreyDhyani) [inline development/select-dropdown-app/components/select-dropdown/ForwardRefSelectDropdown/ForwardRefSelectDropdown.jsx:137]: Remove this comment

File: development/select-dropdown-app/components/select-dropdown/ForwardRefSelectDropdown/ForwardRefSelectDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        enableSearch
+        refresh={refresher}
+      />
+    </div>
+  );
+};
+
+export default ForwardRefInputGroup;
+
+// optionsRenderer={(
```
Comment (ShreyDhyani) [inline development/select-dropdown-app/components/select-dropdown/ForwardRefSelectDropdown/ForwardRefSelectDropdown.tsx:143]: This too.

File: development/select-dropdown-app/pages/select-dropdown/index.mdx [hunk 4]
```diff
@@ -6,7 +6,7 @@ author: "Shrey Dhyani"
 
 import HeadlessSelectSample from "../../components/select-dropdown/HeadlessSelectSample/HeadlessSelectSample.tsx"
 import ReuseSelect from "../../components/select-dropdown/ReuseSelect/ReuseSelect.tsx"
-
+import ForwardRefSelectDropdown from "../../components/select-dropdown/ForwardRefSelectDropdown/ForwardRefSelectDropdown.jsx"
```
Comment (ShreyDhyani) [inline development/select-dropdown-app/pages/select-dropdown/index.mdx:9]: Use tsx file here

**Review by ShreyDhyani — CHANGES_REQUESTED** [review]
@Ayush2020012016 Pls make the requested changes

**Review by Ayush2020012016 — COMMENTED** [review]
removed the logs and comments.

**Comment by Ayush2020012016** [comment]
@ShreyDhyani please review this and please merge. thankyou

**Review by ShreyDhyani — APPROVED** [review]

**Pull request merged**